### ISSUE TITLE
Custom output location/localhost

### ISSUE THREAD

**Issue by Wheemer** [xteve-project/xTeVe#179]
Hello;

I would like to request a custom output filename and location. I am using your software on Libreelec and would love to have it drop a guide.xml into the data directory of my TVHeadend server. So that I do not have to make a cron to do that same work. It would make your software become a seamless integration.

Also it would be nice to have a localhost override for the image caching. It currently defaults to the remote url I am admin accessing the box from. Even though I have that in the hosts they do not show within kodi.

Thanks. Love your software.

**Comment by mar-mei** [comment]
You can write yourself a script to copy the files to another location. The links to the images are based on the last HTTP request, but can be cheated with the xTeVe API. The topic with the links has already been discussed here several times. Take a look at the old issues.

**Comment by Wheemer** [comment]
Yes I realize I can use a script to copy the file, but with every single change you do in xteve it creates a new file. So rather than relying on a timed cron job it would be nice for it to just copy into a designated folder also.

I have turned of caching which fixes my image issue for now. I would rather not have to use a workaround. I made these requests in hopes of making xteve setup and integration easier for the next guy.

**Comment by mar-mei** [comment]
The two files xteve.m3u and xteve.xml are only temporary files, normally these should also be loaded via the xTeVe web server.
Maybe it helps, you can change the default path for the whole configuration folder.
```
xteve -config /my/new/folder/
```

There were two considerations for the BASE URL at the time.
1. xTeVe recognizes the request and then adapts it automatically
2. Each user enters the BASE URL himself.

I decided on the first solution back then, I think it's easier for most people.
